### PR TITLE
Add Member field to ThreadMember struct

### PR DESCRIPTION
**Pull request by ikafly144**
Apply DiscordAPI changes
https://discord.com/developers/docs/change-log#thread-member-details-and-pagination

### CODE REVIEW

**Comment by ikafly144** [comment]
There was already another PR #1314 
I overlooked it. This pull request should be closed

**Comment by ikafly144** [comment]
Stupid I made a mistake in my PR to be closed

File: structs.go [hunk 1]
```diff
@@ -528,6 +528,8 @@ type ThreadMember struct {
 	JoinTimestamp time.Time `json:"join_timestamp"`
 	// Any user-thread settings, currently only used for notifications
 	Flags int
+	// Additional information about the user
+	Member *Member
```
Comment (armsnyder) [inline structs.go:532]: This needs a `json` tag.

**Comment by FedorLap2006** [comment]
Fixed a few things myself, hope you don't mind.
Thanks for the contribution!

**Pull request merged**